docs/source/models/causes/neonatal/neonatal_encephalopathy.rst

```diff
@@ -134,10 +134,10 @@ unit time.
         is determined in the
         intrapartum step of the :ref:`pregnancy model
         <other_models_pregnancy_closed_cohort_mncnh>`)
-    * - neonate didn't die of encephalopathy
-      - The child simulant survived for the first 28 days of life
+    * - neonate did not die of encephalopathy
+      - The child simulant did not die of encephalopathy within the first 28 days of life
     * - neonate died of encephalopathy
-      - The child simulant died within the first 28 days of life
+      - The child simulant died of encephalopathy within the first 28 days of life
```

Comment:
Same comment as on the preterm page, that we probably need to split this into early neonatal and late neonatal instead of lumping all 28 days together.

Reply:
I'm going to leave these for now, and see if the engineers have a preferred change.  If they are confusing people too much, maybe I'll just delete them.